django/contrib/admin/utils.py

```diff
@@ -222,6 +222,11 @@ def nested(self, format_callback=None):
         roots = []
         for root in self.edges.get(None, ()):
             roots.extend(self._nested(root, seen, format_callback))
+        # All values are not stored by key None in dict self.edges
+        for key in self.edges:
+            if str(key) != "None":
```

Comment:
I understand but `dict` keys are not like JavaScript objects, they don't have to be string as long as they are hashable. Here you can use `key is not None` and avoid the `str()` conversion. Give it a try locally.